web/src/environments/environment.ts

```diff
@@ -12,4 +12,6 @@ export const environment: Config = {
     storageBucket: '',
     messagingSenderId: '',
   },
+
+  tracking: 'UA-60804146-5',
```

Comment:
I think this file can have an empty string?

Reply:
done